docs/content/en/docs/getting-started/local-environment/_index.md

```diff
@@ -71,11 +71,23 @@ The configuration specification is divided into two sections:
 
 For full EKS Anywhere configuration reference for a VMware vSphere cluster and explanation on each parameter in the configuration generated above refer vSphere configuration
 
+1. Generate a curated-packages config
+   {{% alert title="Note" color="primary" %}}
+   * It is *optional* to install the curated packages as part of the cluster creation and `eksctl anywhere` version should be `v0.9.0` or later.
+   * Post-creation installation and detailed package configurations can be found [here.]({{< relref "../../tasks/packages" >}})
+   {{% /alert %}}
+   Example shows how to install two packages `flux` and `harbor` from the [curated package list]({{< relref "../../reference/packagespec" >}}).
```

Comment:
```suggestion
   Example shows how to install two packages `harbor` from the [curated package list]({{< relref "../../reference/packagespec" >}}).
```

Reply:
We aren't supporting flux as far as I know. Same comment applies to prod section below